pkgs/top-level/all-packages.nix

```diff
@@ -26755,7 +26755,9 @@ in
 
   nlojet = callPackage ../development/libraries/physics/nlojet { };
 
-  pythia = callPackage ../development/libraries/physics/pythia { };
+  pythia = callPackage ../development/libraries/physics/pythia {
+    hepmc = hepmc2;
```

Comment:
Why do you lock ``hepmc = hepmc2`` if ``pythia`` seems to support ``hepmc3`` (i.e. has configure flags for hepmc3)?

Reply:
Have not had time to test `hepmc3` yet. I am using the `sacrifice` interface which only works with `hepmc2`. There is no real rush to switch now.

Reply:
Maybe add a comment to that effect about why the version is restricted

Reply:
It's not restricted, just that the default is unchanged. `pythia.override { hepmc=hepmc3; }` will compile fine.